src/quicklist.c

```diff
@@ -902,23 +1013,37 @@ REDIS_STATIC void _quicklistInsert(quicklist *quicklist, quicklistEntry *entry,
         }
     }
 
+    if (unlikely(isLargeElement(sz))) {
+        if (QL_NODE_IS_PLAIN(node) || (at_tail && after) || (at_head && !after)) {
+            __quicklistInsertPlainNode(quicklist, node, value, sz, after);
+        } else {
+            quicklistDecompressNodeForUse(node);
+            new_node = _quicklistSplitNode(node, entry->offset, after);
+            entry_node = __quicklistCreatePlainNode(value, sz);
```

Comment:
`entry_node` only used here, and the definition can be moved here.
```suggestion
            quicklistNode *entry_node = __quicklistCreatePlainNode(value, sz);
```